bluechi.spec.in

```diff
@@ -110,61 +112,61 @@ Requires(postun): policycoreutils-python-utils
 %global selinuxtype	targeted
 
 %description selinux
-SELinux policy associated with the hirte and hirte-agent daemons
+SELinux policy associated with the bluechi and bluechi-agent daemons
 
 %files selinux
-%{_datadir}/selinux/devel/include/services/hirte.if
-%{_datadir}/selinux/packages/hirte.pp.bz2
-%{_mandir}/man8/hirte*selinux.*
+%{_datadir}/selinux/devel/include/services/bluechi.if
+%{_datadir}/selinux/packages/bluechi.pp.bz2
+%{_mandir}/man8/bluechi*selinux.*
 
 %post selinux
-%selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/hirte.pp.bz2
-restorecon -R %{_bindir}/hirte* &> /dev/null || :
-semanage port -a -t hirte_port_t -p udp 842 || true
-semanage port -a -t hirte_port_t -p tcp 842 || true
+%selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/bluechi.pp.bz2
+restorecon -R %{_bindir}/bluechi* &> /dev/null || :
+semanage port -a -t bluechi_port_t -p udp 842 || true
+semanage port -a -t bluechi_port_t -p tcp 842 || true
 
 %postun selinux
 if [ $1 -eq 0 ]; then
-   %selinux_modules_uninstall -s %{selinuxtype} hirte
+   %selinux_modules_uninstall -s %{selinuxtype} bluechi
    semanage port -d -p udp 842 || true
    semanage port -d -p tcp 842 || true
-   restorecon -R %{_bindir}/hirte* &> /dev/null || :
+   restorecon -R %{_bindir}/bluechi* &> /dev/null || :
 fi
 
 
 %package ctl
-Summary:  Hirte service controller command line tool
+Summary:  BlueChi service controller command line tool
 Requires: %{name} = %{version}-%{release}
 
 %description ctl
-Hirte is a systemd service controller for multi-nodes environements with a
+BlueChi is a systemd service controller for multi-nodes environements with a
 predefined number of nodes and with a focus on highly regulated environment
 such as those requiring functional safety (for example in cars).
 This package contains the service controller command line tool.
 
 %files ctl
 %doc README.md
 %license LICENSE
-%{_bindir}/hirtectl
-%{_mandir}/man1/hirtectl.*
+%{_bindir}/bluechictl
+%{_mandir}/man1/bluechictl.*
 
 
-%package -n python3-pyhirte
-Summary: Python bindings for Hirte
+%package -n python3-bluechi
+Summary: Python bindings for BlueChi
```

Comment:
This changes from `pyhirte`to `bluechi`. not `pybluechi`. I think this is fine, just checking to see that this was the intention?

Reply:
Yes, most of packages in PIP today are named without `py` prefix

Reply:
I've also mentioned that in the rename google doc file